libmscore/element.h

```diff
@@ -127,7 +127,7 @@ class EditData {
       bool vRaster                     { false };
 
       int key                          { 0     };
-      Qt::KeyboardModifiers modifiers  { 0     };
+      Qt::KeyboardModifiers modifiers  { /*0*/ }; // '0' initialized via default constructor, doing it here too results in compiler warning with Qt 5.15
```

Comment:
Actually my intention was to make it `{}` exactly to avoid  the need for adding comments. But if you believe this construction deserves an extra comment that would be OK as well.

Reply:
To me that construct looks strange enough to warrant an extra comment ;-)

Reply:
OK, let's leave it here then